Sources/AblyAssetTrackingPublisher/WorkerQueue/PublisherWorkerQueueProperties.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import AblyAssetTrackingInternal
+
+internal class PublisherWorkerQueueProperties: WorkerQueueProperties
```

Comment:
`WorkerQueue`’s documentation for `PropertiesType` says "this param must have value semantics". As currently implemented, this class has reference semantics. The easiest way to solve this is by making it a `struct` instead of a `class`. (You’ll also be able to then remove the `init` and instead just write `public var isStopped = false`.)

Reply:
fd7bf5c